Makefile

```diff
@@ -77,6 +78,17 @@ test:
 	sed -i '1d' coverage.txt
 	sed -i '1s/^/mode: atomic\n/' coverage.txt
 
+integration-test:
+	@echo "Building operator integration tests"
+	@cd test/integration_test && go test -tags integrationtest -v -c -o px-operator.test
```

Comment:
Keep this `operator.test` instead of `px-operator.test`. We don't want to have portworx specific Makefile for now.

Reply:
Done